contracts/interfaces/IIBCStore.sol

```diff
@@ -2,7 +2,7 @@
 pragma solidity ^0.8.28;
 
 /// @title IBC Store Interface
-/// @dev Non-view functions can only be called by owner.
+/// @notice The public interface for IBC Store
```

Comment:
These notices doesn't really add any value. It just states what the code is directly saying already. It seems like we should give a bit more info on what the purpose of each interface is. If we think of this as the documentation that an integrator would see, it should be more clear what the purpose and use is.

Reply:
I won't comment on each interface where we have this, but it's a general thing for many of our notice's.